pkg/nvpci/mock.go

```diff
@@ -55,8 +55,13 @@ func (m *MockNvpci) Cleanup() {
 	os.RemoveAll(m.pciDevicesRoot)
 }
 
+type sriovConfig struct {
+	totalVFs int
+	numVFs   int
+}
```

Comment:
If we do what I suggest below, this is not necessary anymore and `SriovInfo` can be used in its place.

Reply:
changed.

Reply:
This struct still seems to be defined (just not used anywhere).